packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -207,4 +217,15 @@ contract SystemConfig is OwnableUpgradeable, Semver {
             sstore(slot, _unsafeBlockSigner)
         }
     }
+
+    /**
+     * @notice Returns the minimum gas limit allowed by the system. If the L2
+     *         gas limit is set to a value smaller than this, then it is
+     *         possible for a block to be produced that uses more gas than what
+     *         is allowed on L2, resulting in a liveness failure.
```

Comment:
Is that what would happen, or would deposit blocks be rejected or invalid? I honestly don't know.

Reply:
According to @trianglesphere it would cause the chain to halt. I don't believe we have test coverage of this case

Reply:
How difficult would it be to add a test for this case in `op-e2e`? Is this something we could do prior to mainnet launch, or okay to put in the backlog in your opinion?